GTG/gtk/editor/editor.py

```diff
@@ -52,6 +53,7 @@ def __init__(self,
         vmanager is the view manager
         thisisnew is True when a new task is created and opened
         '''
+        self.store = None
```

Comment:
You can initiate the store directly here (one less condition in the code bellow).

For extra points, declare the store directly in .ui file and then just do self.builder.get_object()